pipelines/nextflow/modules/genome_metadata/compare_genome_stats.nf

```diff
@@ -19,18 +19,14 @@ process COMPARE_GENOME_STATS {
     label "local"
 
     input:
-        tuple val(db), path(ncbi_stats, stageAs: "ncbi_stats.json"), path(core_stats, stageAs: "core_stats.json")
+        tuple val(db), path(ncbi_stats, stageAs: "ncbi_stats.json"),
+            path(core_stats, stageAs: "core_stats.json")
 
     output:
         tuple val(db), path("diff_stats.json")
 
     script:
```

Comment:
shell?

Reply:
That should be changed in a separate PR (but it definitely needs to be changed).